Revert "doc: Remove instructions on how to authenticate using account… (#312)
codacy/codacy-coverage-reporter: docs/index.md

1.  Set up an API token to allow Codacy Coverage Reporter to authenticate on Codacy.
    {: id="authenticate"}

>     -   **If you're setting up coverage for one repository**, obtain the [project API Token](/repositories-configure/integrations/project-api/) from the page **Integrations** in your Codacy repository settings.

Reviewer: ```suggestion
    -   **If you're setting up coverage for one repository**, obtain the [project API Token](../codacy-api/api-tokens/#project-api-tokens) from the page **Integrations** in your Codacy repository settings.
```

        Then, set the following environment variable to specify your project API Token:

        ```bash
        export CODACY_PROJECT_TOKEN=<your project API Token>
        ```

>     -   **If you're setting up and automating coverage for multiple repositories**, obtain an [account API Token](https://docs.codacy.com/related-tools/api-tokens/) from the page **Access management** in your Codacy account settings.

Reviewer: ```suggestion
    -   **If you're setting up and automating coverage for multiple repositories**, obtain an [account API Token](../codacy-api/api-tokens/#account-api-tokens) from the page **Access management** in your Codacy account settings.
```

        Then, set the following environment variables to specify the account API Token, the username associated with the account API token, and the repository for which you're uploading the coverage information:

        ```bash
        export CODACY_API_TOKEN=<your account API Token>

>         export CODACY_USERNAME=<your account username>

Reviewer: ```suggestion
        export ???=<your account provider>
        export CODACY_USERNAME=<your account username>
```
